client/src/App.js

```diff
@@ -28,11 +37,13 @@ function App() {
             <Link to='/'>Cleanny</Link>
           </Typography>
           {
-            isLoggedIn ? (
-              <Button color='inherit' onClick={logOut}>Wyloguj</Button>
-            ) : ([
-              <Button color='inherit'><Link to='sign-in'>Zaloguj</Link></Button>,
-              <Button color='inherit'><Link to='sign-up'>Zarejestruj</Link></Button>
+            isLoggedIn ? ([
+              <Button color='inherit' startIcon={<ShoppingCartIcon />}><Link to='orders'>Zamówienia</Link></Button>,
```

Comment:
UI po polsku? Myślałem, że ustaliliśmy że wszystko po angielsku. Nazwy usług też są po ang.

Reply:
Znaczy się: w tym przypadku to Kacper napisał to w języku polskim. Już tak to było. Ale jeśli chcesz, to mogę to ogarnąć w języku angielskim